README.md

```diff
@@ -9,32 +41,69 @@ The plugin uses the native libraries [libphonenumber](https://github.com/google/
 | libphonenumber | `8.12.52` |
 | PhoneNumberKit | `3.4.4`   |
 
-## Usage
+## Use
+
+### Setup
 
-### Parsing
+No especific setup required: only install the dependency and use :)
+
+### Basic use
 
-Parse a phone number with region prefix.
 
 ```dart
 String springFieldUSASimple = '+14175555470';
+
+// Parsing
 PhoneNumber phoneNumber = await PhoneNumberUtil().parse(springFieldUSASimple);
+
+// Validate
+bool isValid = await plugin.validate(springFieldUSASimple);
+
+// Format
+RegionInfo region = RegionInfo('US', 1);
+String formatted = await PhoneNumberUtil().format(springFieldUSASimple, region.code); // +1 (417) 555-5470
 ```
 
+See [pud.dev example tab](https://pub.dev/packages/phone_number/example) or [github code](example/lib/main.dart) for more details
```

Comment:
I think this should be simple as: 

See [example](https://github.com/flutter-form-builder-ecosystem/phone_number/tree/main/example) for more details.

Reply:
This is the same text that I use in all other packages